SonarQube.MSBuild.Tasks/MergeRulesets.cs

```diff
@@ -18,7 +18,7 @@ namespace SonarQube.MSBuild.Tasks
     /// MSBuild task to merge rulesets by adding "Include" references to a
     /// specified ruleset
     /// </summary>
-    public class MergeRulesets : Task
+    public class MergeRuleSets : Task
```

Comment:
I changed the casing of "Ruleset" to "RuleSet" to be consistent with the common targets.